cmd/context/utils.go

```diff
@@ -224,7 +224,7 @@ func LoadStackWithContext(ctx context.Context, name, namespace string, stackPath
 }
 
 // LoadContextFromPath initializes the okteto context taking into account command flags and manifest namespace/context fields
-func LoadContextFromPath(ctx context.Context, namespace, k8sContext, path string) error {
+func LoadContextFromPath(ctx context.Context, namespace, k8sContext, path string, defaultCtxOpts *Options) error {
```

Comment:
Why do we need a pointer for `defaultCtxOpts`? I see we always pass a value. We could just pass the value

Reply:
no need to be pointer actually, the code is much better with a value type, changed in e0b7d6b40317993841a2b6226fdf6125a18d56af

thanks!